internal/cypress/docker/docker.go

```diff
@@ -254,6 +263,17 @@ func (handler *Handler) StartContainer(ctx context.Context, c cypress.Project) (
 	return &container, nil
 }
 
+// copyTestFiles copies the files within the container.
+func copyTestFiles(ctx context.Context, handler *Handler, containerID string, files []string, pDir string) error {
+	for _, file := range files {
+		log.Info().Str("from", file).Str("to", pDir).Msg("File copied")
```

Comment:
@FriggaHel should log level here be debug?

Reply:
@vrunoa I used the same level than used for mounting files (https://github.com/saucelabs/saucectl/blob/master/internal/cypress/docker/docker.go#L279).

I also think it may be kept as it allows to quickly view what is going on. This part being the "more obscure" one, I think it's better to keep light on what is injected in the container.